psi4/src/psi4/libfock/PKmanagers.cc

```diff
@@ -370,6 +370,7 @@ void PKManager::form_D_vec(std::vector<SharedMatrix> D, std::vector<SharedMatrix
 
     D_ = D;
     all_sym_ = true;
+    symmetric_.clear();
```

Comment:
@dgasmith Sorry for my late answer. Yes, it was a problem that this vector was not being cleared. Thanks for the correction !